include/aikido/planner/World.hpp

```diff
@@ -7,9 +7,22 @@
 namespace aikido {
 namespace planner {
 
+/// A Kinematic world that contains a set of skeletons.
+/// It is expected that worlds do not share the same skeletons.
 class World
 {
 public:
+  // Encapsulates the state of the World.
+  struct State
```

Comment:
Should `State` also have some notion of the `Skeleton`s it contains? Maybe the `Skeleton` names?

Reply:
One option is to index `configurations` by `Skeleton` name, i.e. switch from an `std::vector` to an `std::unordered_map`.